docs/meshstack.metadata-tags.md

```diff
@@ -181,16 +181,25 @@ Every platform might have different limitations about the tag names and values,
+| Tag Key                       | Description                                                                                                                                                         |
+| ----------------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `${projectIdentifier}`        | The meshProject identifier                                                                                                                                          |
+| `${workspaceIdentifier}`      | The meshProject's workspace identifier                                                                                                                              |
+| `${projectName}`              | The meshProject display name                                                                                                                                        |
+| `${workspaceName}`            | The meshProject's workspace name                                                                                                                                    |
+| `${landingzone}`              | The name of the applied [Landing Zone](meshcloud.landing-zones.md). It contains `no-landingzone` in case the meshProject does not have a Landing Zone applied       |
+| `${landingzoneIdentifier}`    | The identifier of the applied [Landing Zone](meshcloud.landing-zones.md). It contains `no-landingzone` in case the meshProject does not have a Landing Zone applied |
+| `${paymentIdentifier}`        | The identifier of the payment method that is assigned to the meshProject                                                                                            |
+| `${paymentName}`              | The display name of the payment method that is assigned to the meshProject                                                                                          |
+| `${paymentExpirationDate}`    | The expiration date of the payment method that is assigned to the meshProject. It contains `none` if no expiration date is set for the payment method               |
+| `${paymentAmount}`            | The amounf of the assigned payment method. It is `none` if no payment method is set                                                                                 |
+| `${contactMail}`              | Contains the owners email for this workspace. It is `none` if no owner is set                                                                                       |
+| `${ownerUsername}`            | The username of the owner of this meshProjects's workspace. It is `none` if no owner is set                                                                         |
```

Comment:
```suggestion
| `${ownerUsername}`            | The username of the workspace owner of this meshProjects's workspace. It is `none` if no owner is set                                                                         |
```

Reply:
I better remove the `meshProject` term than also from the existing lines.